elide-core/src/main/java/com/yahoo/elide/jsonapi/models/Relationship.java

```diff
@@ -80,6 +81,8 @@ public Relationship(@JsonProperty("links") Map<String, String> links,
                     res.add(resource.toPersistentResource(requestScope));
                 } catch (ForbiddenAccessException e) {
                     //skip resource
+                } catch (java.lang.NullPointerException e) {
```

Comment:
Why throwing with the full pkg path? Is there another NullPointerException class used in this class?